packages/tabs-react/src/Tabs.tsx

```diff
@@ -0,0 +1,187 @@
+export const Tab = React.forwardRef<HTMLButtonElement, TabProps>((props, ref) => (
+    <button className="jkl-tab" role="tab" type="button" ref={ref} {...props} />
+));
+
+interface TabListProps {
+    children: ReactNode;
+    "aria-label": string;
+}
+
+interface TabListInjectedProps {
+    index: number;
+    setIndex: React.Dispatch<React.SetStateAction<number>>;
+    tabIDs: string[];
+    tabPanelIDs: string[];
+}
+
+/**
+ * Konteiner for elementer av typen Tab. Ikke funksjonell utenfor et Tabs element.
+ *
+ * Docs: https://jokul.fremtind.no/komponenter/tabs
+ */
+export const TabList = ({ children, ...injected }: TabListProps) => {
+    // props injected by Tabs
+    const { index, setIndex, tabIDs, tabPanelIDs, ...props } = injected as TabListProps & TabListInjectedProps;
+
+    const [tabsRect, setTabsRect] = useState<DOMRect>();
+    const [activeRect, setActiveRect] = useState<DOMRect>();
+
+    const tabsRef = useRef<HTMLDivElement>(null);
+    const activeRef = useRef<HTMLButtonElement>(null);
+
+    useLayoutEffect(() => {
+        setTabsRect(tabsRef.current?.getBoundingClientRect());
+        setActiveRect(activeRef.current?.getBoundingClientRect());
+    }, [index]);
+
+    const keyDownHandler = useCallback((event: React.KeyboardEvent<HTMLButtonElement>) => {
+        if (event.key === "ArrowLeft") {
+            const current = event.currentTarget;
+            const prev = event.currentTarget.previousSibling;
+
+            if (prev) {
+                (prev as HTMLElement).focus();
+            } else {
+                (current.parentElement?.lastChild?.previousSibling as HTMLElement).focus();
+            }
+        }
+
+        if (event.key === "ArrowRight") {
+            const current = event.currentTarget;
+            const next = event.currentTarget.nextSibling;
+
+            // skip the focus indicator element
+            if (next && next.nodeName === "BUTTON") {
+                (next as HTMLElement).focus();
+            } else {
+                (current.parentElement?.firstChild as HTMLElement).focus();
+            }
+        }
+    }, []);
+
+    return (
+        <div role="tablist" className="jkl-tablist" ref={tabsRef} {...props}>
+            {React.Children.map(children, (tab, tabIndex) => {
+                const isActive = index === tabIndex;
+
+                return React.isValidElement(tab)
+                    ? React.cloneElement(tab, {
+                          onKeyDown: keyDownHandler,
+                          tabIndex: isActive ? undefined : -1,
+                          ref: isActive ? activeRef : undefined,
+                          onClick: () => setIndex(tabIndex),
+                          "aria-selected": isActive,
+                          "aria-controls": tabPanelIDs[tabIndex],
+                          id: tabIDs[tabIndex],
+                      })
+                    : null;
+            })}
+
+            <span
+                className="jkl-tab__indicator"
```

Comment:
```suggestion
                className="jkl-tablist__indicator"
```
Må også endes i scss-fila i så fall